static/src/javascripts/projects/common/modules/experiments/tests/spacefinder-okr-2-images-loaded.ts

```diff
@@ -0,0 +1,21 @@
+import type { ABTest } from '@guardian/ab-core';
+import { noop } from '../../../../../lib/noop';
+
+export const spacefinderOkr2ImagesLoaded: ABTest = {
+	id: 'SpacefinderOkr2ImagesLoaded',
+	author: 'Simon Byford (@simonbyford)',
+	start: '2022-02-14',
+	expiry: '2022-02-28',
+	audience: 20 / 100,
+	audienceOffset: 20 / 100,
+	audienceCriteria: 'All pageviews',
+	successMeasure:
+		'Fixing the bug leads to an increase in inline programmatic revenue per 1000 pageviews',
+	description:
+		"Check whether fixing spacefinder's ability to detect when images have loaded leads to an increase in inline programmatic revenue per 1000 pageviews",
+	variants: [
+		{ id: 'control', test: noop },
+		{ id: 'variant', test: noop },
+	],
+	canRun: () => true,
```

Comment:
Can / should we change this to article pages only? (I could have sworn we'd done this recently but can't find the example now 🤔)

Reply:
Yeah tricky one. We tried something like this before with the previous test (`SpacefinderOkr1FilterNearby`):

https://github.com/guardian/dotcom-rendering/compare/f795b77c3b04de251921ed2bdf44bc614f9b4e69..8f0ad778f6618e395aeff594d7cceca59f350671

But it ended up causing problems in chromatic, where `window.guardian.config.page` wasn't being set for some reason. It felt like a problem with the test environment rather than the code itself, but coding around it (or just ignoring the failure) didn't feel great so we removed the check, having established the filtering could be done in the analysis phase anyway. If we reintroduced the condition, we should also dig deeper into why it breaks chromatic.

One argument for leaving it unfiltered would be consistency with the last test?

Reply:
Yen has confirmed that we can filter out non-article pageviews during the experiment analysis.
I've created a ticket to investigate the chromatic issue [here](https://trello.com/c/TDuj8MfT/1800-investigate-chromatic-failure-when-accessing-guardianconfig-in-abtestcanrun)